cmd/kube-proxy/app/server_others.go

```diff
@@ -175,15 +175,15 @@ func newProxyServer(
 		serviceEventHandler = proxierIPTables
 		endpointsEventHandler = proxierIPTables
 		// No turning back. Remove artifacts that might still exist from the userspace Proxier.
-		klog.V(0).Info("Tearing down inactive rules.")
+		klog.V(0).Info("tearing down inactive rules.")
 		// TODO this has side effects that should only happen when Run() is invoked.
```

Comment:
The clean up logic here is a bit confusing and could use a clean up IMO. I can try looking into this in a follow-up PR

Reply:
oops, got this confused with error messages, fixed!